BuildHelpers/Public/Invoke-Git.ps1

```diff
@@ -77,14 +77,14 @@
             }
             $true
         })]
-        [string]$GitPath = 'git'
+        [string]$GitPath
```

Comment:
The hard coded git.exe we are searching for makes this default value unnecessary.